srcpkgs/apache/template

```diff
@@ -34,7 +34,7 @@ license="Apache-2.0"
 homepage="https://httpd.apache.org/"
 changelog="https://www.apache.org/dist/httpd/CHANGES_2.4"
 distfiles="https://dlcdn.apache.org/httpd/httpd-${version}.tar.bz2"
```

Comment:
I don't know if it's appropriate for this package, but `https://archive.apache.org/dist/httpd/` does not delete outdated versions which would help for local testing when revbumping (as the distfiles are not cached locally if they have not been used earlier).